src/screen/EnhanceScreen_recovery.java

```diff
@@ -0,0 +1,112 @@
+package screen;
+
+import java.awt.event.KeyEvent;
+
+import engine.Cooldown;
+import engine.Core;
+import engine.SoundEffect;
+
+public class EnhanceScreen_recovery extends Screen {
+    /** Milliseconds between changes in user selection. */
+    private static final int SELECTION_TIME = 200;
+    /** Time between changes in user selection. */
+    private Cooldown selectionCooldown;
+    /** For selection moving sound */
+    private SoundEffect soundEffect;
```

Comment:
Codacy Static Code Analysis — 3 new issues (0 max.) of at least minor severity.

2-3. `MINOR` `Code Style` 'VARIABLE_DEF' should be separated from previous line.